services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/application.py

```diff
@@ -155,15 +155,16 @@ def create_app():
     app.add_exception_handler(BaseDynamicSidecarError, http_error_handler)
 
     # EVENTS ---------------------
-    app_state = AppState(app)
 
     async def _on_startup() -> None:
+        app_state = AppState(app)
```

Comment:
does this change something?

Reply:
It avoids validating the shared_store when it is not in place. This will get called after the handler that setups the shared_store and attaches it to the app is called.